packages/zoltan2/test/connectivity/meshData/bigrectangle-borders.quad.msh

```diff
@@ -0,0 +1,21 @@
+big rectangle
```

Comment:
I like using this line as a description of the test.

Reply:
After generating a few of these by hand, I made a boundary-edge generator, which is why many of the descriptions weren't very helpful.